lib/rust/ensogl/core/Cargo.toml

```diff
@@ -25,9 +25,13 @@ enso-shapely = { path = "../../shapely" }
 enso-shortcuts = { path = "../../shortcuts" }
 enso-types = { path = "../../types" }
 enso-web = { path = "../../web" }
+# FIXME: This derive should not be in "app", since it is part of the core theme system. Likely the whole theme system
+# should be moved to its own dedicated crate.
+ensogl-derive-theme = { path = "../app/theme/derive" }
```

Comment:
FYI The topic is much greater; just the entire "core" part of ensogl should be split into crates.